recipe/conda_build_config.yaml

```
@@ -1,5 +1,9 @@
-# Please consult conda-forge/core before doing this
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - "10.13"                # [osx and x86_64]
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - "10.13"                # [osx and x86_64]
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+  - vs                         # [win]
+c_stdlib_version:
+  - 2.17                       # [linux and x86_64]
+  - 2.17                       # [linux and not x86_64]
+  - 10.13                      # [osx and x86_64]
+  - 10.13                      # [osx and arm64]
```

Have you tried this and rerendered? 
```suggestion
  - 11.0                      # [osx and x86_64]
  - 11.0                      # [osx and arm64]
```

---

I believe we can delete this config file. I added it back before the 10.13 SDK was the channel standard. 10.13 is now the channel standard, so it is not necessary to explicitly set the SDK version (unless kokkos now requires a SDK later than 10.13).